docs/calamari/CommunityCollatorsProgram.md

```diff
@@ -0,0 +1,284 @@
+# 🧩  Community Collators Program
+
+## Requirements
+
+1. Hardware Requirement
+
+    a. If running on your own machine, hardware requirement, internet bandwidth
+    
+    ```jsx
+    cpu:8 cores
+    memory: 32 GB of ram
+    disk space: 100 GB of disk space
+    recommended region: EU or US
+    ```
+    
+    b. If running on AWS, instance type and the location guide
+    
+    ```jsx
+    AWS instance: Ubuntu 20.04 (ami-id = ami-04505e74c0741db8d)
+    disk space: r5ad.xlarge
+    recommended region: ca-central, eu-west, eu-central, eu-north, us-east
+    ```
+    
+    NOTE: 
+    
+    - A calamari collator utilizes up to 27gb of ram on a 32gb system.
+    - A calamari collator utilizes up to 25% of cpu on a 24 core system.
+    - A reliable internet connection as well as power supply which should remain uninterrupted to keep local machine **permanently online.**
+2. Bonding requirement: `400_000`KMA (To be changed) at least. Make sure your account has more than `400_000`KMA in your free balance. 
+
+## Deployment
+
+### Tips:
+
+1. The node can be deployed by native binary or docker, it depends on your preference.
+    - [Native](#deploy-a-collator-on-ubuntu-using-native-binary)
+    - [Docker](#deploy-a-collator-on-ubuntu-using-docker)
+2. We provide a recent snapshot which can shorten the time on catching up calamari mainnet.
+
+### Deploy a collator on Ubuntu using native binary
+
+We recommend run on Ubuntu **`20.04 LTS`** :
+1. Get latest manta binary from manta repo: [https://github.com/Manta-Network/Manta/releases](https://github.com/Manta-Network/Manta/releases)
+    
+- Get calamari genesis file: [https://github.com/Manta-Network/Manta/blob/manta/genesis/calamari-genesis.json](https://github.com/Manta-Network/Manta/blob/manta/genesis/calamari-genesis.json)
+- Get relaychain(kusama) genesis: [https://github.com/paritytech/polkadot/blob/master/node/service/res/kusama.json](https://github.com/paritytech/polkadot/blob/master/node/service/res/kusama.json)
+
+2. Start a collator by the following command with RPC port enabled. Please specify `your_db_path`.
+
+    > If you want to start collator with snapshot, download snapshot from [here](#download-snapshot),
+    > follow this [section](#native) to deploy node with snapshot.
+
+    ```bash
+    manta --base-path your_db_path \    
+    --name your_collator_name \
+    --chain calamari-genesis.json \
+    --rpc-cors 'all' \
+    --port 30333 \
+    --rpc-port 9933 \
+    --rpc-methods=Unsafe \
+    --force-authoring \
+    --collator \
+    -- \
+    --chain kusama.json
```

Comment:
Seems strange to me that peer discovery on the relaychain would work without 30334 opened, but I remember you said you tested it.
Just wondering, did you test it with all nodes specified in the genesis_config as bootnodes or did you have a discovered node on the relaychain as well?